### PR TITLE
PCHR-1800: Manager calendar styles

### PR DESCRIPTION
**Pull request by deb1990**
This PR contains the styles for Manager Calendar static page.
manager leave hr17

### CODE REVIEW

File: civihr_default_theme/scss/components/_manager_calendar.scss [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    display: inline-block;
+    margin: 0 10px;
+    width: calc(16.4% - 20px);
+    vertical-align: top;
+  }
+}
+
+th.chr_manager_calendar__name,
+td.chr_manager_calendar__name {
+  font-size: 12px !important;
```
Comment (dhavalsharma) [inline civihr_default_theme/scss/components/_manager_calendar.scss]: can font sizes be in scss variables?

File: civihr_default_theme/scss/components/_manager_calendar.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  line-height: 100%;
+  padding: 0 0 0 20px !important;
```
Comment (dhavalsharma) [inline civihr_default_theme/scss/components/_manager_calendar.scss]: why is !important required here?

File: civihr_default_theme/scss/components/_manager_calendar.scss [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  text-align: left;
+  vertical-align: middle !important;
```
Comment (dhavalsharma) [inline civihr_default_theme/scss/components/_manager_calendar.scss]: why is !important required here?

**Review by dhavalsharma — CHANGES_REQUESTED** [review]

File: civihr_default_theme/scss/components/_manager_calendar.scss (hunk 2 above)
Comment (deb1990) [reply civihr_default_theme/scss/components/_manager_calendar.scss]: To override already existing style
```css
.chr_leave-calendar__days {

  > th {
    @media (min-width: $screen-lg-min) {
      padding: 10px 0 !important;
    }
  }
}
```

File: civihr_default_theme/scss/components/_manager_calendar.scss (hunk 3 above)
Comment (deb1990) [reply civihr_default_theme/scss/components/_manager_calendar.scss]: To override
```css
.table>thead>tr>th {
    vertical-align: bottom;
}
```

**Review by dhavalsharma — APPROVED** [review]

**Pull request merged**